src/plugins/highcharts/renderer/helpers/config/options.js

```diff
@@ -216,15 +216,7 @@ const options = {
             statesForLine,
         ),
         areaspline: first,
-        bar: Object.assign(
-            {
-                borderWidth: 0,
-                pointWidth: 4,
```

Comment:
remove this props due to it always overwrites the default width of bars, thereby forcing user to set pointWidth value null (which is incorrect) or the fixed value.